os/arch/arm/src/common/up_internal.h

```diff
@@ -278,7 +278,7 @@ EXTERN uint32_t _edata;			/* End+1 of .data */
 EXTERN uint32_t _sbss;			/* Start of .bss */
 EXTERN uint32_t _ebss;			/* End+1 of .bss */
 EXTERN uint32_t _sidle_stack;		/* Start of idle stack */
-EXTERN uint32_t __sint_heap_start;	/* Start of heap in internal RAM region */
+EXTERN uint32_t _sint_heap_start;	/* Start of heap in internal RAM region */
 EXTERN uint32_t __sext_heap_start;	/* Start of heap in external RAM region */
```

Comment:
_s as a prefix to any variable indicates that the variables denotes the starting address.
For ex: - _sidle_stack denotes the start of the idle stack.
An extra "_start" at the end is unnecessary, i.e. _sidle_stack_start is not needed.

Similarly, _sint_heap is enough to denote the start of the internal heap
The extra "_start" at the end is unnecessary.